src/test/java/subway/domain/LineTest.java

```diff
@@ -38,4 +47,42 @@ void updateLine() {
                 () -> assertThat(노선_신분당선.getColor()).isEqualTo(녹색)
         );
     }
+
+    @DisplayName("노선의 구간을 추가한다.")
+    @Test
+    void addSection() {
+
+        final Section 첫번째_구간 = new Section(1L, 강남역, 양재역, 10);
+        final Line 노선_신분당선 = new Line(1L, 신분당선, 빨간색, 구간_신규_등록(첫번째_구간));
+        노선_신분당선.addSection(양재역, 몽촌토성역, 10);
```

Comment:
```suggestion
        final Section 첫번째_구간 = new Section(1L, 강남역, 양재역, 10);
        final Line 노선_신분당선 = new Line(1L, 신분당선, 빨간색, 구간_신규_등록(첫번째_구간));
        노선_신분당선.addSection(양재역, 몽촌토성역, 10);

        final Station 강남역2 = new Station(4L, "강남역2");
        노선_신분당선.addSection(몽촌토성역, 강남역2, 10);
```

이렇게 Section 을 추가하려고 하니 오류가 발생하네요 😢 
<img width="712" alt="스크린샷 2023-01-31 오후 7 10 54" src="https://user-images.githubusercontent.com/17218212/215731296-97813a97-1ad9-425e-8e31-c5cda7f008b1.png">

Reply:
앗 수정하겠습니다!